src/schema/slack/functions/invite_user_to_channel_test.ts

```diff
@@ -71,11 +56,14 @@ Deno.test("InviteUserToChannel can be used as a built-in function in a workflow
     title: "Test InviteUserToChannel",
     description: "This is a generated test to test InviteUserToChannel",
   });
-  testWorkflow.addStep(InviteUserToChannel, {});
+  testWorkflow.addStep(InviteUserToChannel, {
+    channel_ids: "test",
+    user_ids: "test",
```

Comment:
Same thing here re: passing strings in but the inputs should be an array.

Reply:
Yes this is part of the generation script behavior, If I remember correctly the types here do not actually get validated so the generation script does not implement the proper types

I originally implemented this behavior, if we want to add it in it should be possible

Reply:
I'll leave that up to @shapirone to make a call on!